docs/guide/reporters/json.md

```diff
@@ -0,0 +1,121 @@
+WDIO JSON Reporter
```

Comment:
We need a preamble here, see how other files do that: https://raw.githubusercontent.com/webdriverio/webdriverio/master/docs/guide/reporters/dot.md. See the `index` property, make sure you order it between the last  reporter and before the custom reporter